src/main/java/frc/robot/subsystems/shooter/ShooterSS.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.components.TrigonTalonFX;
+import frc.robot.constants.RobotConstants.ShooterConstants;
+import frc.robot.subsystems.TestableSubsystem;
+import frc.robot.utilities.EncoderConversions;
+
+public class ShooterSS extends SubsystemBase implements TestableSubsystem {
+    private final TrigonTalonFX masterMotor;
+
+    public ShooterSS() {
+        masterMotor = ShooterConstants.LEFT_MOTOR;
```

Comment:
blank line after line 14, as this writing addresses the next two lines as a separate section of the code